source/system/Character.ooc

```diff
@@ -6,7 +6,7 @@
  * of the MIT license.  See the LICENSE file for details.
  */
 
-include stdlib
+include stdlib, stdint
```

Comment:
This is to make our fixed-width integers work now right? Shouldn't `stdint` be included in the use file under a `windows` version flag? Otherwise, will we need to add this everywhere?